src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,35 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+interface Props {
+  activeTodo: Todo;
+  handleSetActiveTodo: (arg: Todo | undefined) => void;
+}
+
+export const TodoModal: React.FC<Props> = ({
+  activeTodo,
+  handleSetActiveTodo,
+}) => {
+  const [user, setUser] = useState<User | undefined>(undefined);
+
+  useEffect(() => {
+    getUser(activeTodo.userId).then(data => {
+      setUser(data);
+    });
+  }, []);
```

Comment:
```suggestion
  }, [activeTodo.userId]);
```
<img width="548" alt="image" src="https://github.com/mate-academy/react_dynamic-list-of-todos/assets/64750958/dc474006-e952-4db9-80bf-d2c3ecb26b55">